vertx-oracle-client/src/main/java/io/vertx/oracleclient/OracleException.java

```diff
@@ -11,34 +11,21 @@
 
 package io.vertx.oracleclient;
 
-import io.vertx.core.VertxException;
+import io.vertx.sqlclient.DatabaseException;
+
+import java.sql.SQLException;
 
 /**
- * A {@link RuntimeException} thrown when a Reactive Oracle Client error occurs.
+ * The {@link DatabaseException} for Oracle.
  */
-public class OracleException extends VertxException {
-
-  public OracleException(String message) {
-    super(message);
-  }
-
-  public OracleException(String message, Throwable cause) {
-    super(message, cause);
-  }
-
-  public OracleException(Throwable cause) {
-    super(cause);
-  }
-
-  public OracleException(String message, boolean noStackTrace) {
-    super(message, noStackTrace);
-  }
+public class OracleException extends DatabaseException {
 
-  public OracleException(String message, Throwable cause, boolean noStackTrace) {
-    super(message, cause, noStackTrace);
+  public OracleException(String message, int errorCode, String sqlState) {
+    super(message, errorCode, sqlState);
```

Comment:
we should format an error message like we do for `PgException ` and `DB2Exception`

Reply:
I've update other exception classes but not this one because the error messages coming from the Oracle driver already contain a lot of information, including error code and states